base/logger_file.go

```diff
@@ -43,6 +44,8 @@ var (
 
 	belowMinValueFmt = "%s for %v was set to %v which is below the minimum of %v"
 	aboveMaxValueFmt = "%s for %v was set to %v which is above the maximum of %v"
+
+	lumberjackRotationMidfix = `-\d{4}-\d{2}-\d{2}T\d{2}-\d{2}-\d{2}.\d{3}`
```

Comment:
related to suggestion in `getDeletionDirAndRegexp`

```suggestion
	// lumberjack backupTimeFormat = "2006-01-02T15-04-05.000"
	lumberjackRotationMidfix = `-\d{4}-\d{2}-\d{2}T\d{2}-\d{2}-\d{2}.\d{3}`
	// lumberjack compressSuffix = ".gz"
	optionalCompressSuffix = `(\.gz)?`
```